.github/pull_request_template.md

```diff
@@ -8,6 +8,7 @@
 - `yarn test:unit:all` should run without errors or warnings
```

Comment:
# ZAP Errors

@jusdino Could you confirm which of the Medium and High- risk vulnerabilities are false-positives vs. actual issues? For those that are actual issues, mind creating a PR to fix? (If the change is nontrivial or low-risk, feel free to just put it in the backlog instead.)

One that _does_ stand out to me is the Format String Error which returns an Internal Server Error.

Report is here: https://github.com/csg-org/CompactConnect/actions/runs/11483838084

Reply:
Discussed privately, closing.